app/services/accept_offer.rb

```diff
@@ -46,6 +46,6 @@ def application_choices_awaiting_provider_decision
   end
 
   def unconditional_offer?
-    @application_choice.offer&.fetch('conditions', []).blank?
+    @application_choice.offer.conditions.none?
```

Comment:
Can we just use `@application_choice.unconditional_offer?` here?

Reply:
No, because `unconditional_ofer?` returns false unless the application choice is in the recruited state. 

Something to tackle as part of the refactoring task.

Reply:
Ok, it's very confusing as is so we should address at some point